pyccel/commands/lambdify.py

```diff
@@ -38,14 +38,14 @@ def lambdify(expr : sp.Expr, args : 'dict[sp.Symbol, str]', *, result_type : str
         expressions do not always evaluate to the expected type. For
         example if the SymPy expression simplifies to 0 then the default
         type will be int even if the arguments are floats.
-    templates : dict[str,list[str]], optional
+    templates : dict[str, list[str]], optional
         A description of any templates that should be added to the
         function. The keys are the symbols which can be used as type
         specifiers, the values are a list of the type annotations which
         are valid types for the symbol. See
         <https://github.com/pyccel/pyccel/blob/devel/docs/templates.md>
         for more details.
-    collect_result_in_arg : bool, False
+    use_out : bool, False
```

Comment:
According to NumPyDoc's [style guide](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) we should write `default=False` and not just `False`.

Reply:
Done [bc57116](https://github.com/pyccel/pyccel/pull/1867/commits/bc5711641cb5d1731ea97a0af297b99062787592)